net/SslSocket.hpp

```diff
@@ -48,12 +48,12 @@ class SslStreamSocket final : public StreamSocket
             throw std::runtime_error("Failed to create SSL.");
         }
 
-        if (!hostname.empty())
+        if (!hostname().empty())
```

Comment:
Do we want this functional change instead of using host ?

Reply:
`hostname()` returns the member that is the `host` argument. Not a functional change (as the test should attest).

Reply:
If you're happy with that; I worry about the system hostname() function but as you like.